extensions/amp-story/1.0/amp-story.css

```diff
@@ -957,6 +957,10 @@ amp-story-page.i-amphtml-story-desktop-panels .i-amphtml-story-interactive-compo
   border: none !important;
 }
 
+body:not(.amp-mode-keyboard-active) .i-amphtml-story-screen-reader-back-button {
```

Comment:
What's the reasoning that `.amp-mode-keyboard-active` works for Talkback? Will this always be the case?

Reply:
Good idea. TalkBack and VoiceOver add an outline to the currently focused element. 
To the best of my knowledge this will always be the case since it's a core feature of mobile screen readers UX.

Reply:
Some tablets and/or phones have keyboards, should we hide the outline all the time? There's the aria-label for audio feedback, is this enough?

Reply:
Good point. Since this button is meant to be an affordance to mobile screen readers which add outlines, it feels okay to just have audio feedback.

Reply:
> Followup question: why was this button on screen in the first place? Can it be clicked/tapped?

It needs to be rendered on screen so TalkBack can highlight it and show it to the user. 
This is to follow TalkBacks expected UX.